post_installation_configuration/aws-compute-edge-tasks.adoc

```diff
@@ -0,0 +1,73 @@
+:_content-type: ASSEMBLY
+[id="aws-compute-edge-tasks"]
+= AWS Local Zone tasks
+include::_attributes/common-attributes.adoc[]
+:context: aws-compute-edge-tasks
+
+toc::[]
+
+After installing {product-title} on Amazon Web Services (AWS), you can further configure AWS Local Zones and an edge compute pool, so that you can expand and customize your cluster to meet your needs.
+
+// Extend existing clusters to use AWS Local Zones
+include::modules/post-install-edge-aws-extend-cluster.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+* For more information about AWS Local Zones, the supported instances types, and services, see link:https://aws.amazon.com/about-aws/global-infrastructure/localzones/features/[AWS Local Zones features] in the AWS documentation 
+
+// About the Edge Compute Pool
+include::modules/edge-machine-pools-aws-local-zones.adoc[leveloffset=+2]
+
+[id="post-install-extend-existing-to-local-zones-mtu"]
+=== Changing the Cluster Network MTU to support AWS Local Zones subnets
+
+You might need to change the maximum transmission unit (MTU) value for the cluster
+network, so that your cluster infrastructure can support Local Zone subnets. 
+
+// About the cluster MTU
+include::modules/nw-cluster-mtu-change-about.adoc[leveloffset=+3]
+
+// Changing the cluster MTU
+include::modules/nw-cluster-mtu-change.adoc[leveloffset=+3]
+
+// Opting in to AWS Local Zones
+include::modules/installation-aws-add-local-zone-locations.adoc[leveloffset=+2]
+
+// Create a Local Zone subnet
+include::modules/post-install-existing-local-zone-subnet.adoc[leveloffset=+2]
+
+// Creating a subnet in AWS Local Zones
+include::modules/installation-creating-aws-subnet-localzone.adoc[leveloffset=+3]
+
+// CloudFormation template for the subnet that uses AWS Local Zones
+include::modules/installation-cloudformation-subnet-localzone.adoc[leveloffset=+3]
+
+// Creating a machine set manifest for AWS Local Zones node
+include::modules/post-install-edge-aws-extend-machineset.adoc[leveloffset=+2]
+
+// Sample YAML for a compute machine set custom resource on AWS
+include::modules/machineset-yaml-aws.adoc[leveloffset=+3]
+
+// Creating a compute machine set
+include::modules/machineset-creating.adoc[leveloffset=+3]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../installing/installing_aws/installing-aws-localzone.adoc[Installing a cluster using AWS Local Zones]
```

Comment:
Not a big deal, but the friendly reference text isn't the same as the module if you want them to match.